tools/Raven.Migrator/CommandLineApp.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using Microsoft.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+using Raven.Migrator.CosmosDB;
+using Raven.Migrator.MongoDB;
+
+namespace Raven.Migrator
+{
+    internal static class CommandLineApp
+    {
+        private const string HelpOptionString = "-h | -? | --help";
+
+        private static readonly string MongoDbDescription =
+            "Connect to MongoDB, pass configuration as JSON" + Environment.NewLine +
+            "   Command - available commands: databases, collections, export" + Environment.NewLine +
+            "   ConnectionString - MongoDB connection string" + Environment.NewLine +
+            "   DatbaseName - the database name, applicaple only for the collections and export commands" + Environment.NewLine +
+            "   MigrateGridFS - (Optional) migrate GridFS that is associated with the provided database" + Environment.NewLine +
+            "   CollectionsToMigrate - (Optional) a dictionary of collections to rename during the export";
+
+        private static readonly string CosmosDbDescription =
```

Comment:
I think that this MS command line supports multi-level commands and multi-level help? Talk with @gregolsky about that.

Reply:
Have a look in https://github.com/ravendb/ravendb/blob/v4.1/tools/rvn/CommandLineApp.cs

Reply:
I have a JSON over there. Not sure how this can work.

Reply:
Then I suggest using 'nameof'